### ISSUE TITLE
The result psnr is different from the log of github.

### ISSUE THREAD

**Issue by xiaoloutongxue** [DevKiHyun/SRCNN-Tensorflow#1]
Dear Sir, I have ran your code which is a good project. However,  the result of mine are different  from your log file.In my result, the PSNR of bicubic is 33.65, and the PSNR of SRCNN is always below 36.50. Do you know why? Look forward to your reply，thank you for your help.

**Comment by DevKiHyun** [comment]
Thanks xiaoloutongxue. I also found this problem. Your test set's psnr value is lower than my log because I fixed the psnr calculation function after I uploaded the log file. As a result, the average psnr value of the test set recorded in my log is an invalid value. So don't worry. If you want to know that the training has been done correctly, it is a good idea to first visually judge the result images. I will upload demo.py and test.py about SRCNN model. Thank you.

**Comment by DevKiHyun** [comment]
I updated log.txt. Thanks
